aikau/src/main/resources/alfresco/layout/FixedHeaderFooter.js

```diff
@@ -277,8 +270,8 @@ define(["alfresco/core/ProcessWidgets",
        *
        * @instance
        */
-      _resize: function alfresco_layout_FixedHeaderFooter___resize() {
-         if (this.recalculateAutoHeightOnResize === true)
+      onResize: function alfresco_layout_FixedHeaderFooter__onResize() {
```

Comment:
This naming suggests actions to occur after resize has happened, but it looks like it's actually doing the resize? Can it be called `resize()`, or even `doResize()` if there's a naming clash?